src/fluree/db/conn/file.cljc

```diff
@@ -203,38 +246,42 @@
   #?@(:clj
       [full-text/IndexConnection
        (open-storage [conn network dbid lang]
-         (throw (ex-info "File connection does not support full text operations."
-                         {:status 500 :error :db/unexpected-error})))]))
+                     (throw (ex-info "File connection does not support full text operations."
+                                     {:status 500 :error :db/unexpected-error})))]))
 
 
 (defn connect
   "Create a new file system connection."
   [{:keys [context did local-read local-write parallelism storage-path publish-path] :as opts}]
-  (let [conn-id  (str (random-uuid))
-        commit   (connection-commit storage-path)
-        read     (connection-read storage-path)
-        write    (connection-write storage-path)
-        exists?  (connection-exists? storage-path)
-        rename   (connection-rename storage-path)
-        push     (connection-push publish-path)
-        state    (state-machine/blank-state)
-        close-fn (fn [] (log/info (str "File Connection " conn-id " Closed")))]
-    ;; TODO - need to set up monitor loops for async chans
-    (map->FileConnection {:id          conn-id
-                          :transactor? false
-                          :context     context
-                          :did         did
-                          :local-read  local-read
-                          :local-write local-write
-                          :read        read
-                          :write       write
-                          :commit      commit
-                          :push        push
-                          :exists?     exists?
-                          :rename      rename
-                          :parallelism parallelism
-                          :msg-in-ch   (async/chan)
-                          :msg-out-ch  (async/chan)
-                          :close       close-fn
-                          :memory      true
-                          :state       state})))
+  (go-try
```

Comment:
What is the `go-try` needed for?

Reply:
All of the different `conn` methods should now return a channel instead of a Connection. Brian changed the signature for the ipfs conn, this just updates the file conn to match.

Reply:
I see. Might it be better to just use `go` if we're not using the added exception handling stuff in `go-try`? Or maybe some of these called fns could throw? Fine as-is if so.

Reply:
It doesn't look like there's anything that could go wrong in setting up a file conn, so I changed it to `go`.